.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -64,4 +62,4 @@ Otherwise, write a comment and it will be done for you.
 -->
 
 - [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
-- [ ] Add the "ready to merge" label **if the basics are fulfilled** and no further pushes are planned by you.
+- [ ] Add the "ready to merge" label **if everything is done** and no further pushes are planned by you.
```

Comment:
Again I'd really prefer, if we moved to using draft PRs instead of labels.

Reply:
Draft PRs are not prohibited.

Reply:
Even if we don't promote them (personally I don't see why not), I would state that using the "Draft PR" feature is an alternative to the labels, i.e. if a PR is already a draft it shouldn't need to be labelled "work in progress". Doing both seems redundant.